src/views/shared/SecurityBeacon.js

```diff
@@ -41,11 +41,14 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
       el.removeAttr('aria-describedby');
       el.find('.accessibility-text').text(imgDescription);
       el.css('background-image', 'url(' + _.escape(imgSrc) + ')');
-      return;
```

Comment:
nit: this seems to me doesn't affect the method, so we could probably leave it as it is in case someone in the future adds another ```if``` block